### PR TITLE
Add GitLab to social

### PR DESCRIPTION
**Pull request by HackingGate**
The SVG icons comes from [https://github.com/encharm/Font-Awesome-SVG-PNG](https://github.com/encharm/Font-Awesome-SVG-PNG).

- Support multiple instances

Example of _config.yml:
```yml
gitlab:
  - username: jekyll
    instance: gitlab.com
  - username: jekyll
    instance: gitlab.freedesktop.org
```

Looks:
GitLab-jekyll

Closes #325

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,6 +221,7 @@ minima:
     twitter: jekyllrb
     github: jekyll
     stackoverflow: "11111"
+    gitlab: jekyll
```
Comment (DirtyF) [inline README.md]: oops

Comment (DirtyF) [inline README.md]:
```suggestion
```

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
 .jekyll-cache
 .sass-cache
-Gemfile.lock
 _site
+
+# Bundler
 *.gem
+.bundle
+Gemfile.lock
+vendor/bundle
```
Comment (ashmaroli) [inline .gitignore]: @DirtyF  Let us handle this in a separate dedicated PR..

File: README.md [hunk 3]
```diff
@@ -238,6 +239,12 @@ minima:
      - username: jekyll2
        instance: example.com
 
+    gitlab:
+     - username: jekyll
+       instance: example.com
+     - username: jekyll2
+       instance: example.com
```
Comment (ashmaroli) [inline README.md]: Do we intend to support an array of instances or a single string username..??

File: assets/minima-social-icons.svg [hunk 4]
```diff
@@ -43,4 +43,8 @@
...
   <path transform="scale(0.039)" d="M80 205v51.2l10.3-.4c8.9-.3 10.8-.7 14.8-3 7.8-4.6 7.9-4.9 7.9-48.4 0-42.4 0-42.4-7.3-47.2-3.6-2.4-5.1-2.7-14.9-3.1l-10.8-.3V205z"/>
 </symbol>
+
+<symbol id="gitlab" fill-rule="evenodd" clip-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="1.414" viewBox="0 0 1792 1792">
+        <path d="M104 706l792 1015-868-630q-18-13-25-34.5t0-42.5l101-308zm462 0h660l-330 1015zm-198-612l198 612h-462l198-612q8-23 33-23t33 23zm1320 612l101 308q7 21 0 42.5t-25 34.5l-868 630 792-1015zm0 0h-462l198-612q8-23 33-23t33 23z"/>
```
Comment (ashmaroli) [inline assets/minima-social-icons.svg]: Heavy indentation here..

File: README.md (hunk 3 above)
Comment (DirtyF) [reply README.md]: It's the latter.

**Comment by ashmaroli** [comment]
I couldn't find any documentation on GitLab instances..

**Comment by DirtyF** [comment]
You can host your instance of GitLab (https://about.gitlab.com/pricing/#self-managed), so you can have multiple instances.

**Comment by ashmaroli** [comment]
Thank you @HackingGate 
@jekyllbot: merge +minor

**Comment by DirtyF** [comment]
@jekyllbot: merge+minor

**Comment by p2635** [comment]
Is this in minima 2.5.1? How can I use this? I tried to do something like this but it didn't work.

    gitlab:
     - username: jekyll

**Comment by hszhakka** [comment]
> Is this in minima 2.5.1? How can I use this? I tried to do something like this but it didn't work.
> 
> ```
> gitlab:
>  - username: jekyll
> ```

This will be included in Minima 3.0, I guess. However, the release date looks remote. I suggest you use `jekyll-remote-theme` to import the `master` branch so that you can use this feature. You can refer to this [post](https://hszhakka.github.io/2020/01/20/launch-my-blog.html).
